### DISCUSSION TITLE
How to add remotion in existing react project ( for dynamic videos as it is shown on https://remotion.dev/)

### DISCUSSION THREAD

**Discussion by sandeep-asts** [remotion-dev/remotion#664] [Q&A] [answered]
<img width="1010" alt="Screenshot 2021-10-23 at 2 06 48 PM" src="https://user-images.githubusercontent.com/89834130/138549655-4c1fbedc-7b17-498c-ae19-c12f0624e18c.png">

I want to add this feature to my existing react project, Idea is similar to the above dynamic video feature on the remotion home page. 

Providing UI to enter the name and generate a birthday greeting video :)

Thanks in advance

**Comment by arthurdenner** [comment, marked as answer]
Hi! Please have a look at the Player package:
https://remotion.dev/docs/player

**Reply by sandeep-asts** [reply]
Thanks @arthurdenner 

I am able to integrate the remotion player and pass the existing react component.
